tls/s2n_cipher_preferences.c

```diff
@@ -884,6 +884,19 @@ struct {
     { .version=NULL, .preferences=NULL, .ecc_extension_required=0, .pq_kem_extension_required=0}
 };
 
+struct {
+    const char *version_str;
+    int version_int;
+} protocol_version_lookup [] = {
+    { .version_str="S2N_SSLv2", .version_int=20},
```

Comment:
nitpick: If this list were reversed would we match the protocol quicker? I assume the normal case is to set 1.2 or 1.3 as the minimum.

Reply:
Sure done!